content/docs/user-guide/gems/reference/aws/aws-gamelift/flexmatch/scripting.md

```diff
@@ -0,0 +1,129 @@
+---
+linkTitle: FlexMatch Scripting
+title: FlexMatch Scripting
+description: Learn about Script Canvas nodes for player matchmaking in the AWS GameLift Gem in Open 3D Engine (O3DE).
+toc: true
+weight: 400
+---
+
+
+The **AWS GameLift** provides Script Canvas nodes that make requests against Amazon GameLift to start, stop and accept matchmaking requests.
+
+There are synchronous and asynchronous versions for each action. Asynchronous nodes perform their operation in the AZ JobFunction and finish at some point in the future. These operations communicate through AWS HTTPS requests or over the network. Each asynchronous node has a corresponding notification handler node.
```

Comment:
Whats the different about communication via AWS Https request or over the network?

Reply:
This line is copied from the existing session management scripting page. Will check with @onecent1101 for the meaning of it and make the change accordingly. It's not clear to me as well.

Reply:
Worked with Vincent and doc. I've rephrased this line.